sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/spark_job_entry_mixin.py

```diff
@@ -39,8 +39,13 @@ def entry(self, value: Union[Dict[str, str], SparkJobEntry, None]):
         self._entry = value
 
     def _validate_entry(self):
+        if self.entry is None:
```

Comment:
do we have a test case covering this scenario?

Reply:
@wangchao1230 

we need wait for doris pr: https://msdata.visualstudio.com/Vienna/_git/sdk-cli-v2/pullrequest/927066?_a=files&path=/src/mldesigner/mldesigner/_generate/_generators/_constants.py, should we add a test and skip it for now?

And it's a test with mldesigner, should we put it in sdk-cli-v2 repro or this repro

Reply:
I see. Let's do it there then.